src/components/ErrorPage.tsx

```diff
@@ -1,59 +1,75 @@
-import { Box, Button, Center, Flex, Heading, Link } from "@chakra-ui/react";
+import { Box, Button, Center, Link, VStack } from "@chakra-ui/react";
+
+import { useOffboardingModal } from "./Offboarding/useOffboardingModal";
+import { ModalContentWrapper } from "./Onboarding/ModalContentWrapper";
+import { downloadBackupFile } from "./Onboarding/restoreBackupFile/utils";
+import { NoticeIcon, ReloadIcon } from "../assets/icons";
+import BackgroundImage from "../assets/onboarding/background_image.png";
+import colors from "../style/colors";
 
 const feedbackEmailBodyTemplate =
   "What is it about? (if a bug report please consider including your account address) %0A PLEASE FILL %0A%0A What is the feedback? %0A PLEASE FILL";
 
-// TODO: prepare for Beta release properly
 export const ErrorPage: React.FC = () => {
+  const { modalElement: OffboardingModal, onOpen: onOpenOffboardingModal } = useOffboardingModal();
+
   const onRefresh = () => {
     window.location.reload();
   };
 
-  const onBackup = () => {
-    const jsonString = `data:text/json;chatset=utf-8,${encodeURIComponent(
-      JSON.stringify(window.localStorage)
-    )}`;
+  return (
+    <Center height="100vh" padding="60px" backgroundImage={BackgroundImage} backgroundSize="cover">
+      <Box
+        width="100%"
+        maxWidth="460px"
+        height="100%"
+        maxHeight="480px"
+        padding="40px"
+        background={colors.gray[900]}
+        border="1px"
+        borderColor={colors.gray[700]}
+        borderRadius="8px"
+        boxShadow="0px 0px 30px rgba(0, 0, 0, 0.30)"
+      >
+        <ModalContentWrapper
+          icon={<NoticeIcon />}
+          subtitle="Please refresh the app or use one of the following options:"
```

Comment:
the titles have smaller text sizes than they have to and the header block is narrower on the design, please also verify that the spacing between it and the first button is correct

Reply:
In my opinion we should use the same size of header & subtitle for all similar modals. However, Figma contains different font sizes for different modals atm.

Issue with the header block width is also present with the other modals (e.g. for onboarding)

I've sent a question to Bahador regarding this as well

Reply:
According to Bahador:

> Please follow the same logic as the other modals (e.g. onboarding). So use the same font size, and the same width as other modals.

The description width should be constant & smaller than modal context's width